app/controllers/api/v1/items_merchant_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::ItemsMerchantController < ApplicationController
```

Comment:
New class looks good and functional. May have to include error serializer in this.